riff-raff/app/views/restrictions/list.scala.html

```diff
@@ -0,0 +1,62 @@
+@import _root_.restrictions.RestrictionConfig
+@import _root_.restrictions.RestrictionChecker
+@import helper.CSRF
+@(configs: Seq[RestrictionConfig])(implicit request: Security.AuthenticatedRequest[AnyContent, com.gu.googleauth.UserIdentity])
+
+@main("Restrictions", request){
+    <h2>Restrictions</h2>
+    <hr/>
+    <div class="alert alert-info">
+        <p><strong>Disable or limit deploys of projects</strong></p>
+        <p>Restrictions are principally designed to be used when there is a temporary issue with a project or environment such
+        that deploying would be dangerous or potentially degrade the product. In addition they can be used when
+        infrastructure security could be compromised by allowing any user to rollback or deploy a branch of their
+        choice.</p>
+    </div>
+    <p><a class="btn btn-primary" href="@routes.Restrictions.form()"><i class="glyphicon glyphicon-plus glyphicon glyphicon-white"></i> Add new restriction</a></p>
+    <div class="content">
+    @if(configs.isEmpty) {
+        <div class="alert alert-warning"><strong>No restrictions.</strong></div>
+    } else {
+        <table class="table table-condensed">
+            <thead>
+                <tr>
+                    <th>Last edited</th>
+                    <th>Project Name</th>
+                    <th>Stage</th>
+                    <th>Locked</th>
+                    <th></th>
+                    <th></th>
```

Comment:
If this `<th>` doesn't mean anything by itself you could colspan it with the previous one as the header for "actions".

:minor_point: